analysis/measures_definition.py

```diff
@@ -73,13 +73,16 @@
 
 # Demographic variables and other patient characteristics
 # Define variable that checks if a patients is registered at the start date
-has_registration = practice_registrations.for_patient_on(
+registration = practice_registrations.for_patient_on(
     INTERVAL.start_date
 ).exists_for_patient()
 
 age = patients.age_on(INTERVAL.start_date)
 age_greater_equal_65 = age >= 65
 
+# Define population to be registered and above 18 years old
+has_registration = registration & (age > 18)
```

Comment:
`has_registration` here is not just registration but also age. was there a reason you changed the names here?